cmd/github-ldap-user-group-creator/main.go

```diff
@@ -359,24 +363,27 @@ func validate(group *userv1.Group) error {
 	return nil
 }
 
-func UpsertGroup(ctx context.Context, client ctrlruntimeclient.Client, group *userv1.Group) (created bool, err error) {
```

Comment:
I've repurposed `created` because it was ignored anyway. The method is not used elsewhere, so I made it private, too.

Reply:
It was copied from the existing work. 😉